raco/algebra.py

```diff
@@ -1221,14 +1221,25 @@ def __eq__(self, other):
 
 class OrderBy(UnaryOperator):
 
-    """ Logical Sort operator
-    """
+    """ Logical OrderBy operator"""
 
     def __init__(self, input=None, sort_columns=None, ascending=None):
         UnaryOperator.__init__(self, input)
         self.sort_columns = sort_columns
         self.ascending = ascending
 
+    def __eq__(self, other):
+        return UnaryOperator.__eq__(self, other) and \
+            self.sort_columns == other.sort_columns and \
+            self.ascending == other.ascending
+
+    def __repr__(self):
+        return "{op}({inp!r}, {scol!r}, {asc!r})".format(
+            op=self.opname(),
+            inp=self.input,
+            scol=list(self.sort_columns),
```

Comment:
`repr()` is used to represent query plans as executable Python code (this format is used by the `myrial` script via `plan_from_repr()`, while JSON format is used by MyriaX). Specifically, the output of `repr()` for each logical operator class must be a legal Python constructor call. Therefore, all arguments to `__init__` must appear (in the same order) as format string arguments in `__repr__`.

I think my question was unclear: what I was really asking was why you assign `scol=list(self.sort_columns)` instead of `scol=self.sort_columns`.